apps/platforms/mac/sarafi/safari.py

```diff
@@ -0,0 +1,75 @@
+from talon import ctrl, ui, Module, Context, actions, clip, app
+
+ctx = Context()
+mod = Module()
+apps = mod.apps
+mod.apps.safari = """
+os: mac
+and app.bundle: com.apple.Safari
+"""
+
+ctx.matches = r"""
+app: safari
+"""
+ctx.tags = ['browser', 'user.tabs']
```

Comment:
i've noticed in a few places above that tags are asserted like this, rather than in the .talon file. i'm just curious if this became the preferred way to do it? i usually do it from the .talon

Reply:
after reviewing the rest of the files i noticed this seems to be part of the migration script when the associated talon file no longer has anything so get deleted